src/test/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesControllerTest.java

```diff
@@ -100,9 +100,38 @@ void testCreatingANewSubmissionIsSuccessful() throws ServiceException {
     }
 
     @Test
-    void testCreatingANewSubmissionIsSuccessfulWithValidation() throws ServiceException {
+    void testCreatingANewUpdateSubmissionIsSuccessfulWithValidation() throws ServiceException {
         setValidationEnabledFeatureFlag(true);
```

Comment:
There was review comment & change to use reflection to set this flag which is now merged to main. You'll need to rebase / merge main & fix conflicts I'm afraid.